build.assets/Makefile

```diff
@@ -381,6 +439,58 @@ enter-root: buildbox
 		-e HOME=$(SRCDIR)/build.assets -w $(SRCDIR) $(BUILDBOX) /bin/bash
 
 #
+# Reconfigure crosstool-NG for the given ARCH via its menuconfig system.
+# e.g.
+# $ make config/buildbox-ng CTNG_ARCH=arm64
+#
+# After saving and exiting in the menuconfig, you should have an updated
+# defconfig file in build.assets/buildbox/ct-ng-configs
+#
+# You need to have a local copy of the configuration container image
+# created with: make buildbox-thirdparty-ctng
+#
+.PHONY: config/buildbox-ng
+config/buildbox-ng:
+	@: $(or $(CTNG_ARCH),$(error CTNG_ARCH variable must be set))
+	@: $(or $(wildcard buildbox/ct-ng-configs/$(CTNG_ARCH).defconfig),\
+		$(error No config for arch $(CTNG_ARCH)))
+	docker run -it --rm \
+		--hostname $(HOSTNAME) \
+		--volume $(shell pwd)/..:/home/teleport \
+		--volume /tmp:/tmp \
```

Comment:
Why do we need to mount /tmp? Can we mount subfolder instead of the whole /tmp?

Reply:
This is what is done for the other buildboxes. It doesn't matter on CI - it will always have a fresh /tmp for each job, but for local development, persisting the go and rust directories (GOPATH and CARGO_HOME which are set to /tmp directories for the buildbox) will make dev easier.

Reply:
In this case I'd use docker "internal caching" if you want to speed up the build. We're doing the same thing in TAG: https://github.com/gravitational/access-graph/blob/main/Dockerfile#L42
Otherwise you may hit weird filesystem related issues as "cannot remove root owned files" or similar.

Reply:
I'm using these caches for the download directory, but this cache is not for the `docker build` but for across `docker run` invocations, mostly to save some time during dev. It's not needed for functionality, so I've removed it for now and I'll look at what makes sense as an optimisation after.